test/integration/IntegrationTests.spec.ts

```diff
@@ -285,10 +267,25 @@ describe("Upload DAR integration test", () => {
         }
     };
 
-    it('should successfully submit and wait for a command', (done) => {
+    it('should be able to upload Dar file', (done) =>{
+        const darToUploadInString = darToUpload.toString('base64');
         withLedgerClient((client) => {
-            client.commandClient.submitAndWait(commands, (error) => {
+            client.packageManagementClient.uploadDarFile({
+                darFile: darToUploadInString
+            });
+
+            client.packageManagementClient.listKnownPackages((error, response) => {
                 expect(error).to.be.null;
+
+                if (response !== undefined) {
```

Comment:
This is an error condition, you're effectively skipping the test if this hold true. It's probably ok to `expect(response).to.not.be.null` before this line.

Reply:
See next commit